src/config/backpack/operations/create.php

```diff
@@ -37,13 +37,15 @@
 
 /**
  * Before saving the entry, how would you like the request to be stripped?
- * - false - fall back to Backpack's default (ONLY save inputs that have fields)
- * - closure - process your own request (example removes all inputs that begin with underscode).
+ * - false - use Backpack's default (ONLY save inputs that have fields)
+ * - closure - custom request processing (quick, but will stop config caching)
```

Comment:
😅 I agree. But we can't remove it now, it's been a feature for 4 months already. Gotta keep it until our next release, otherwise it's just confusing for someone who's used it before...

Reply:
Future-me here. 

Damn it! Yeah, I guess you're right @pxpm ... we shouldn't keep instruct people on how to do something WRONG... 😔 we'd be making more problems here...

Ok I removed that instruction. Good catch!

Reply:
I don't think we **can't** remove it. 

- New users will not get it - fine
- Old users that might have a closure setup ... will still have the closure setup and working - fine
- Old users starting a new project ... read the docblock and adapt? - fine
- Old users starting a new project ... copy from old project? works as previously. - fine 

We are leaving something bad because probably 10% (I might be exagerating here) of devs that overwrote this in the last 4 months may be confused ? It still does not sell to me.

I guess 99% of the 10% that configures this setting `strippedRequest` do it on a per crud basis, because only in crud X and Z they have some extra field they need to handle. 

I don't think the closure on config was used that many times.

This said I rest my case, if this was not enough to change your mind, please mark the conversation as solved and let's move on with our lives 👍

Reply:
No, you're right. I changed my mind and hit "send" 1 min before you did 😅 sorry for the trouble

Reply:
Nice to know I changed your mind while I was thinking and writting about it. We should try that again, maybe this was not just a coincidence 🥳